doc/groups.md

```diff
@@ -1 +1,315 @@
 # Groups Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add Groups](#add) | POST | /groups-add | Add Groups API
+[Edit Groups](#edit) | PUT | /groups-edit/{id} | Edit Groups API
+[View All Groups](#view) | GET | /groups | View Groups API
+[Delete Groups](#delete) | DELETE | /groups-delete/{id} | Delete Groups API
+
+
+### Database
+![](./groups-layer.png)
+
+*For [Add Groups](#add) and [Edit Groups](#edit), you need get aros to set aros_id.* 
+
+## <a name="add"></a>Add Groups
+
+### Endpoint 
+POST /groups-add
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+
```

Comment:
Add header email

Reply:
my bad, sorry for my bad.
for groups module,  there is no filter content by user login, correct?
so we don't need email in header.

Reply:
done, sir